sailfishusermanager.cpp

```diff
@@ -37,6 +37,7 @@ namespace {
 const char *USER_GROUP = "users";
 const char *GROUPS_USER = "USER_GROUPS";
 const auto GROUPS_IDS_FILE = QStringLiteral("/usr/share/sailfish-setup/group.ids");
+const auto USER_UUID_FILE = QStringLiteral("/etc/user_uuid");
```

Comment:
Is there some reason why passwd gecos field is not used for the uuid? That would be more standardized way of storing user data.

Reply:
May be we can do it, for now we store full user name in gecos, we can add custom format for this field. Let me check this.

Reply:
I think we can do it. We add some restrictions on full user name(no comma), is it ok?

Reply:
Yes, sounds good.